File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/clinical/core/inbound/dtos/DocumentSignatureRequestedInput.java
package io.zenwave360.example.clinicaltool.modules.clinical.core.inbound.dtos;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import jakarta.validation.constraints.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;

/**
 * DocumentSignatureRequestedInput.
 */
@lombok.Getter
@lombok.Setter
public class DocumentSignatureRequestedInput implements Serializable {

    @NotNull
    private Long documentInfoId;

    @NotNull
    private Long patientId;
}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/core/inbound/dtos/DocumentSignatureRequestedInput.kt
package io.zenwave360.example.clinicaltool.modules.clinical.core.inbound.dtos

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import jakarta.validation.constraints.*
import java.io.Serializable
import java.math.*
import java.time.*

/** DocumentSignatureRequestedInput. */
data class DocumentSignatureRequestedInput(
    @NotNull val documentInfoId: Long? = null,
    @NotNull val patientId: Long? = null,
) : Serializable {}
